frontend/src/lib/components/InsightCard/InsightCard.tsx

```diff
@@ -33,100 +35,140 @@ export interface InsightCardProps extends React.HTMLAttributes<HTMLDivElement> {
     removeItem: () => void
     /** Callback for refreshing insight. */
     refresh: () => void
+    /** Callback for renaming insight. */
+    rename: () => void
+    /** Callback for duplicating insight. */
+    duplicate: () => void
+    /** Callback for moving insight to another dashboard. */
+    moveTo: (dashboardId: DashboardType['id']) => void
```

Comment:
very nittiest of nits

I'd either move the info in the comments into the method names (`moveTo` becomes `moveInsightToDashboard`) or remove them. I think the context is clear enough without the comment

Reply:
Fair point, I renamed the props to make the comments totally redundant!